### PR TITLE
ticdc: add scale out for kafka changefeed

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add doc about scale out feature for kafka changefeed.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:  N/A
- Other reference link(s): https://github.com/pingcap/tiflow/pull/8100

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@overvenus Could you please invite a tech reviewer? Thanks!

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 将大单表的同步负载水平扩展到多个 TiCDC 节点上
+
+该功能可以解决单个 TiCDC 节点不能及时同步大单表的问题和 TiCDC 节点之间资源（CPU，内存等）消耗出现倾斜的问题。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:239]:
```suggestion
该功能可以解决单个 TiCDC 节点不能及时同步大单表的问题和 TiCDC 节点之间资源（CPU、内存等）消耗出现倾斜的问题。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+配置样例如下所示：
+
+```shell
+[scheduler]
+region-per-span = 50000
+```
+
+> **注意：**
+>
+> v6.6 的 TiCDC 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:252]:
```suggestion
> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+
+## 将大单表的同步负载水平扩展到多个 TiCDC 节点上
+
+该功能可以解决单个 TiCDC 节点不能及时同步大单表的问题和 TiCDC 节点之间资源（CPU，内存等）消耗出现倾斜的问题。
+
+该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。
+
+配置样例如下所示：
+
+```shell
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:245]:
```suggestion
```toml
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@3AceShowHand PTAL

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 将大单表的同步负载水平扩展到多个 TiCDC 节点上
```
Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:237]: 水平扩展大单表的负载到多个 TiCDC 节点上

Comment (3AceShowHand) [reply ticdc/ticdc-sink-to-kafka.md:237]: 水平 -> 横向

File: ticdc/ticdc-sink-to-kafka.md (hunk 1 above)
Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:239]: 资源消耗倾斜 -> 不均匀

Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:239]: 这句话包含了两个场景，建议两个场景用子弹列表分开描述。

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+
+该功能可以解决单个 TiCDC 节点不能及时同步大单表的问题和 TiCDC 节点之间资源（CPU、内存等）消耗出现倾斜的问题。
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:239]:
```suggestion
该功能可以解决以下两个问题：

- 单个 TiCDC 节点不能及时同步大单表的问题
- TiCDC 节点之间资源（CPU、内存等）消耗不均匀的问题。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 6]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 > ```
+
+## 将大单表的同步负载水平扩展到多个 TiCDC 节点上
+
+该功能可以解决以下两个问题：
+
+- 单个 TiCDC 节点不能及时同步大单表的问题
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀的问题。
+
+该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。
```
Comment (3AceShowHand) [inline ticdc/ticdc-sink-to-kafka.md:244]: 使得多个 TiCDC 节点可以同时同步大单表。-> 使得多个 TiCDC 节点，可以同时单张表的数据。

**Review by 3AceShowHand — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 7]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 将大单表的同步负载水平扩展到多个 TiCDC 节点上
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:237]:
```suggestion
## 横向扩展大单表的负载到多个 TiCDC 节点
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 8]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能可以解决以下两个问题：
+
+- 单个 TiCDC 节点不能及时同步大单表的问题
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀的问题。
+
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:243]:
```suggestion
> **注意：**
>
> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。

```

File: ticdc/ticdc-sink-to-kafka.md [hunk 9]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+配置样例如下所示：
+
+```toml
+[scheduler]
+region-per-span = 50000
+```
+
+> **注意：**
+>
+> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:255]:
```suggestion

```

File: ticdc/ticdc-sink-to-kafka.md [hunk 10]
```diff
@@ -233,3 +233,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能可以解决以下两个问题：
+
+- 单个 TiCDC 节点不能及时同步大单表的问题
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀的问题。
+> **注意：**
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:243]:
```suggestion

> **注意：**
```

**Review by Oreoxmt — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 11]
```diff
@@ -233,3 +233,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能可以解决以下两个问题：
+
+- 单个 TiCDC 节点不能及时同步大单表的问题
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀的问题。
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-kafka.md:242]:
```suggestion
- 单个 TiCDC 节点不能及时同步大单表。
- TiCDC 节点之间资源（CPU、内存等）消耗不均匀。
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 12]
```diff
@@ -233,3 +233,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能可以解决以下两个问题：
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-kafka.md:239]: 建议一句话描述下这个功能，再说这个功能的作用。

File: ticdc/ticdc-sink-to-kafka.md [hunk 13]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+
+- 单个 TiCDC 节点不能及时同步大单表。
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀。
+
+> **注意：**
+>
+> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
+
+该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:248]:
```suggestion
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 14]
```diff
@@ -233,3 +233,23 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能可以解决以下两个问题：
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:239]:
```suggestion
该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 15]
```diff
@@ -233,3 +233,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
 > ```
 > {matcher = ['*.*'], dispatcher = "ts", partition = "table"},
 > ```
+
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:239]: @3AceShowHand @shichun-0415 PTAL

File: ticdc/ticdc-sink-to-kafka.md [hunk 16]
```diff
@@ -233,3 +233,22 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+
+- 单个 TiCDC 节点不能及时同步大单表。
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀。
+
+> **注意：**
+>
+> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
+
+
```
Comment (overvenus) [inline ticdc/ticdc-sink-to-kafka.md:248]:
```suggestion

```

File: ticdc/ticdc-sink-to-kafka.md [hunk 17]
```diff
@@ -233,3 +233,20 @@ partition 分发器用 partition = "xxx" 来指定，支持 default、ts、index
...
+## 横向扩展大单表的负载到多个 TiCDC 节点
+
+该功能通过将大单表按 Region 个数切分成多个数据范围，将这些数据范围分布到多个 TiCDC 节点上，使得多个 TiCDC 节点可以同时同步大单表。该功能可以解决以下两个问题：
+
+- 单个 TiCDC 节点不能及时同步大单表。
+- TiCDC 节点之间资源（CPU、内存等）消耗不均匀。
+
+> **注意：**
+>
+> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:246]:
```suggestion
> TiCDC v6.6.0 仅支持在 Kafka 同步任务上开启大单表的水平扩展功能。

```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-sink-to-kafka.md (hunk 15 above)
Comment (shichun-0415) [reply ticdc/ticdc-sink-to-kafka.md:239]: LGTM

**Review by shichun-0415 — APPROVED** [review]
LGTM

**Pull request merged**